packages/cli/src/services/frontend.service.ts

```diff
@@ -379,6 +382,15 @@ export class FrontendService {
 			this.settings.easyAIWorkflowOnboarded = false;
 		}
 
+		// Load AI usage settings
+		try {
+			this.settings.ai.allowSendingParameterValues =
```

Comment:
if we do what I mentioned above with the cache service, we only call the service and behind the scenes we either get the value from the cache or from the db, instead trying to get values from the env variables.